test/Microsoft.AspNetCore.SignalR.Tests/TestClient.cs

```diff
@@ -25,7 +25,7 @@ public class TestClient : IDisposable, IInvocationBinder
 
         public DefaultConnectionContext Connection { get; }
         public Channel<byte[]> Application { get; }
-        public Task Connected => Connection.Metadata.Get<TaskCompletionSource<bool>>("ConnectedTask").Task;
+        public Task Connected => (Connection.Metadata["ConnectedTask"] as TaskCompletionSource<bool>).Task;
```

Comment:
I am saying it should be: `TaskCompletionSource<bool>)Connection.Metadata["ConnectedTask"]` because we don't expect this to be of a different type?